integrations/SnapPixel/util.js

```diff
@@ -14,10 +14,10 @@ const sendEvent = (event, payload) => {
   }
 };
 
-const getCommonEventPayload = (message) => {
+const getCommonEventPayload = (message, deduplicationKey) => {
   let payload = {
     price: parseFloat(get(message, "properties.price")),
-    client_deduplication_id: get(message, "properties.client_deduplication_id"),
+    client_deduplication_id: get(message, `${deduplicationKey || "messageId"}`),
```

Comment:
```suggestion
    client_deduplication_id: get(message, "properties.client_deduplication_id") || get(message, `${deduplicationKey || "messageId"}`),
```
To make it backward compatible.

Reply:
updated